politics/lib/get-home-props.ts

```diff
@@ -0,0 +1,167 @@
+import { Bet } from 'common/bet'
+import { MultiSerializedPoints, SerializedPoint, binAvg } from 'common/chart'
+import { Contract } from 'common/contract'
+import { getMultiBetPoints, getSingleBetPoints } from 'common/contract-params'
+import { fetchLinkPreviews } from 'common/link-preview'
+import {
+  ChartParams,
+  MapContractsDictionary,
+  NH_LINK,
+  presidency2024,
+} from 'common/politics/elections-data'
+import { getBetPoints, getBets } from 'common/supabase/bets'
+import {
+  ChartAnnotation,
+  getChartAnnotations,
+} from 'common/supabase/chart-annotations'
+import { getContractFromSlug } from 'common/supabase/contracts'
+import { SupabaseClient } from 'common/supabase/utils'
+import { unstable_cache } from 'next/cache'
+import { initSupabaseAdmin } from 'web/lib/supabase/admin-db'
+export const REVALIDATE_CONTRACTS_SECONDS = 60
+
+export async function getHomeProps(useUnstableCache: boolean) {
```

Comment:
Only the politics site uses `useUnstableCache`, so you can eliminate that option from both where you got this code, and where it lives now.